src/components/PageHeader.tsx

```diff
@@ -19,9 +19,9 @@ export const PageHeader = ({ title, description, image, lightMode, toggleLightMo
           <Icon name={lightMode ? "SunIcon" : "MoonIcon"} color="dark" size="md" />
         </div>
       </div>
-      <div className="flex flex-col justify-start items-start self-stretch gap-5 lg:w-1/2">
+      <div className="flex flex-col justify-start items-start self-stretch gap-5 @3xl:w-3/4 @6xl:w-1/2">
```

Comment:
@NilsJacobsen Why do we need an additional size here? Do we have to differentiate between two cases? The 3xl and the 6xl width case? In comparison of the other code, this feels like an anomaly, that's why I'm asking.

Reply:
I had the feeling that the header was to shrinked for tablet sizes. Check out the last commit and check tablet and mobile pages in storybook. It's in the page stories.

Reply:
Nice, alright! Because the header is subject to change, this can be merged and revisited in near future.